docs/src/tutorials/dgm.md

```diff
@@ -0,0 +1,74 @@
+## 2D Poisson's equation
```

Comment:
This title doesn't reflect that it's a DGM tutorial.

Reply:
Then it should actually be written as a tutorial, i.e. explain what the DGM is and how to use it.  As it is right now, it is just an example because it doesn't explain anything about DGM, just solves.

Reply:
I'll be more careful with writing tutorials from now on. Sorry for this time. I'll paste the markdown render here for DGM, before pushing it. Also, changing the tutorial code to Burger's equation instead of Poisson's equation because it better fits the definition of PDE being a dynamic system in time and serves as a better example imho.